comet/broker.py

```diff
@@ -488,7 +426,6 @@ def saveDataset(hash, ds, root):
     -H "Content-Type: application/json"
```

Comment:
This example command doesn't seem to have a `roots` key so I think it would actually fail on L431.